plugin/hotswap-agent-spring-plugin/src/main/java/org/hotswap/agent/plugin/spring/getbean/DetachableBeanHolder.java

```diff
@@ -44,16 +44,13 @@ public class DetachableBeanHolder implements Serializable {
     private Object beanFactory;
     private Class<?>[] paramClasses;
     private Object[] paramValues;
-    private static List<WeakReference<DetachableBeanHolder>> beanProxies =
-            Collections.synchronizedList(new ArrayList<WeakReference<DetachableBeanHolder>>());
+    //key: bean name
```

Comment:
instead of put a such simple comment here, I suggest to explain why a map is necessary, especially why the value is a list.